modules/lss-missionDate/missionDate.user.js

```diff
@@ -65,7 +65,7 @@
     if (I18n.locale === 'en') {
       var offset = today.getTimezoneOffset() / 60;
       // Zahl 4, weil EST UTC+4 ist
-      var newOffset = 4 - offset;
+      var newOffset = 5 - offset;
```

Comment:
Bitte Kommentar anpassen - wieso wurde es jetzt von 4 auf 5 gesetzt?

Reply:
Da auch die Timezone in der Winterzeit ist, und daher 1 Stunde mehr ist ( http://www.zeitzonen.de/eastern_standard_time_est_-_usa.html ). Sonst würde er im Einsatzfenster einfach eine Stunde anzeigen, obwohl der Einsatz grad mal 2 min da ist (bei Missionchief)